Marlin/src/feature/bedlevel/ubl/ubl.h

```diff
@@ -264,8 +264,8 @@ class unified_bed_leveling {
         return UBL_Z_RAISE_WHEN_OFF_MESH;
     #endif
 
-    const uint8_t mx = _MIN(cx, (GRID_MAX_POINTS_X) - 2) + 1, my = _MIN(cy, (GRID_MAX_POINTS_Y) - 2) + 1,
-                  x0 = get_mesh_x(cx), x1 = get_mesh_x(cx + 1);
+    const uint8_t mx = _MIN(cx, (GRID_MAX_POINTS_X) - 2) + 1, my = _MIN(cy, (GRID_MAX_POINTS_Y) - 2) + 1;
+    const float x0 = get_mesh_x(cx), x1 = get_mesh_x(cx + 1);
     const float z1 = calc_z0(rx0, x0, z_values[cx][cy], x1, z_values[mx][cy]),
```

Comment:
```suggestion
    const float x0 = get_mesh_x(cx), x1 = get_mesh_x(cx + 1),
                z1 = calc_z0(rx0, x0, z_values[cx][cy], x1, z_values[mx][cy]),
```

Reply:
If it's no major difference, I'd prefer it as is. Initializing variables that way is exactly what led to the bug in the first place, and it was harder to find due to it. Also feels like bad practice to use the variable on the same line as it's initialized, but that might just be me being picky.

Reply:
On second thought, it shouldn't pose any issues